src/fpu/fpu_instructions.h

```diff
@@ -320,20 +298,18 @@ static void FPU_FST_I32(PhysPt addr) {
 
 static void FPU_FST_I64(PhysPt addr)
 {
-	FPU_Reg temp_reg;
-	// If the value was loaded with the FILD/FIST 64-bit memcpy trick,
-	// read the 64-bit integer value instead.
-	if (fpu.regs_memcpy[TOP].has_value()) {
-		temp_reg.ll = fpu.regs_memcpy[TOP].value();
-	} else {
-		double val  = FROUND(fpu.regs[TOP].d);
-		temp_reg.ll = (val <= static_cast<double>(INT64_MAX) &&
-		               val >= static_cast<double>(INT64_MIN))
-		                    ? static_cast<int64_t>(val)
-		                    : LONGTYPE(0x8000000000000000);
+	constexpr double MaxInt64Value = 9223372036854775808.0;
+	auto val_i                     = fpu.regs_memcpy[TOP];
```

Comment:
extremely minor nitpicking: maybe a newline after the constexpr to make tha auto-formatter align this more nicely 😄

Reply:
but it is aligned nicely; look at those matching equals signs!